Formula/postgresql@9.4.rb

```diff
@@ -49,19 +50,6 @@ def install
       args << "--with-tclconfig=#{MacOS.sdk_path}/System/Library/Frameworks/Tcl.framework"
     end
 
-    # As of Xcode/CLT 10.x the Perl headers were moved from /System
```

Comment:
Can you explain why this change was made and perl was added as a dep?

Reply:
Sorry, this PR was a bit blunt! I ended up removing the `perl` dep. After some digging, I figured the actual fix was to define `PG_SYSROOT`:

https://github.com/Homebrew/homebrew-core/blob/a10973b2ef03410ada4f94cc95a7f291cdbb30b6/Formula/postgresql%409.4.rb#L27

I updated the PR's description and the commit message with a short explanation.